src/backend/aspen/workflows/nextstrain_run/run_nextstrain_autorun.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# WDL inputs available through environmental vars:
+# AWS_REGION
+# GENEPI_CONFIG_SECRET_NAME
+# REMOTE_DEV_PREFIX (if set)
+# DEPLOYMENT_STAGE
+
+set -Eeuxo pipefail
+shopt -s inherit_errexit
+
+df 1>&2
+cat /proc/meminfo 1>&2
+
+start_time=$(date +%s)
+
+aws configure set region $AWS_REGION
+
+if [ -n "${BOTO_ENDPOINT_URL-}" ]; then
+  export aws="aws --endpoint-url ${BOTO_ENDPOINT_URL}"
+else
+  export aws="aws"
+fi
```

Comment:
Not relevant to this PR specifically, but I think we need to look for a way to reduce all the copy/paste boilerplate in our wdl and bash wrappers.